### ISSUE TITLE
Should aria-checked be supported on all input elements?

### ISSUE THREAD

**Issue by Epigenetic** [w3c/html-aam#461]
Currently, the spec says that the checked attribute should be mapped to aria-checked on any input element (see https://w3c.github.io/html-aam/#att-checked). Since we have this mapping, it MUST be exposed to user agents. However, the checked state only makes sense in a couple of input types, namely "checkbox" and "radio". Additionally, Chrome and Firefox both do not seem to expose the checked state for other input types. Should this mapping be updated to only apply to the relevant input types?

**Comment by scottaohara** [comment]
hi @Epigenetic.  i'll add some clarifying text to the mapping table.  This is not trying to say that checked is allowed on any input, but rather when it is allowed (e.g., on radios and checkboxes) that it would expose these properties.
